tron/kubernetes.py

```diff
@@ -274,21 +262,9 @@ def handle_event(self, event: Event) -> None:
 
                 self.done()
         except Exception as e:
-            self.log.exception(f"unable to handle an event for id={event_id} for event={str(event)}")
-            # clog here and make sure the message is a string
-            if clog is None:
-                log.debug("Clog logger unavailable. Unable to log event")
-            else:
-                clog.log_line(
-                    "tmp_missed_tronevents",
-                    json.dumps(
-                        {
-                            "event": str(event),
-                            "exception": type(e),
-                            "exception_message": str(e),
-                        }
-                    ),
-                )
+            self.log.exception(
+                f"unable to handle an event for id={event_id} for event={str(event)}, exception type: {type(e)} and exception message: {str(e)}"
```

Comment:
log.exception() will print out the exception below - we can just log the info we want to add on top of the exception itself

Reply:
I added all the info needed in the self.log.exception message, such as event_id, the event that failed, type of the exception and exception message

Reply:
ohh you mean add the extra info in a self.log.info message following the exception message?